### PR TITLE
Add LIP "Introduce certificate generation mechanism"

### PR DESCRIPTION
**Pull request by janhack**

### CODE REVIEW

File: proposals/lip-introduce_certificate_generation_mechanism.md [hunk 1]
```diff
@@ -0,0 +1,843 @@
+```
+LIP: <LIP number>
+Title: Introduce certificate generation mechanism
+Author: Jan Hackfeld <jan.hackfeld@lightcurve.io>
+Discussions-To: https://research.lisk.com/t/introduce-certificate-generation-mechanism/296
+Type: Standards Track
+Created: <YYYY-MM-DD>
+Updated: <YYYY-MM-DD>
+Requires: Introduce BFT module LIP,
+          Introduce Validators module LIP,
```
Comment (ricott1) [inline proposals/lip-introduce_certificate_generation_mechanism.md:10]: LIP0044

File: proposals/lip-introduce_certificate_generation_mechanism.md [hunk 2]
```diff
@@ -0,0 +1,843 @@
...
+          Update block schema and block processing LIP
```
Comment (ricott1) [inline proposals/lip-introduce_certificate_generation_mechanism.md:11]: LIP 0055

File: proposals/lip-introduce_certificate_generation_mechanism.md [hunk 3]
```diff
@@ -0,0 +1,843 @@
...
+```
+
+
+[ccu-lip]: https://research.lisk.com/t/introduce-cross-chain-update-transactions/298
+[bft-module-lip]: https://research.lisk.com/
+[block-header-lip]:  https://research.lisk.com/t/update-block-schema-and-block-processing/293
+[dpos-lip]: https://research.lisk.com/t/define-state-and-state-transitions-of-dpos-module/320
+[poa-lip]: https://research.lisk.com/t/proof-of-authority-validator-selection-mechanism/288
+[unlock-lip]: https://research.lisk.com/t/introduce-unlocking-condition-for-incentivizing-certificate-generation/300
+[validator-module-lip]: https://research.lisk.com/t/introduce-validators-module/317
```
Comment (ricott1) [inline proposals/lip-introduce_certificate_generation_mechanism.md:842]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0044.md

File: proposals/lip-introduce_certificate_generation_mechanism.md [hunk 4]
```diff
@@ -0,0 +1,843 @@
...
+    if aggregateBFTWeight >= lastCertificateThreshold:
+        return eligibleSingleCommits
+    else:
+        return []
+```
+
+
+[ccu-lip]: https://research.lisk.com/t/introduce-cross-chain-update-transactions/298
+[bft-module-lip]: https://research.lisk.com/
+[block-header-lip]:  https://research.lisk.com/t/update-block-schema-and-block-processing/293
```
Comment (ricott1) [inline proposals/lip-introduce_certificate_generation_mechanism.md:838]: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0055.md

Comment (ricott1) [reply proposals/lip-introduce_certificate_generation_mechanism.md:838]: In general many of these links could already be updated

**Review by ricott1 — CHANGES_REQUESTED** [review]
We can already update some links

File: proposals/lip-introduce_certificate_generation_mechanism.md (hunk 3 above)
Comment (janhack) [reply proposals/lip-introduce_certificate_generation_mechanism.md]: Changed.

File: proposals/lip-introduce_certificate_generation_mechanism.md (hunk 4 above)
Comment (janhack) [reply proposals/lip-introduce_certificate_generation_mechanism.md]: Changed.

File: proposals/lip-introduce_certificate_generation_mechanism.md (hunk 1 above)
Comment (janhack) [reply proposals/lip-introduce_certificate_generation_mechanism.md]: Changed.

**Comment by janhack** [comment]
As Alessandro suggested, I updated the links for all LIPs that are already merged.

**Review by MaximeGagnebin — APPROVED** [review]

**Review by ricott1 — APPROVED** [review]

**Review by ishantiw — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]
🚀

**Review by karmacoma — APPROVED** [review]

**Pull request merged**